ocaml/xapi-consts/api_errors.ml

```diff
@@ -13,6 +13,14 @@
  *)
 exception Server_error of string * string list
 
+let ( $ ) f x = f x
```

Comment:
I don't like having this much churn, why can't the collection be autogenerated like in other languages?

Reply:
Other sdks don't use the api_errors. I'll split into its own commit.